src/java.management/share/classes/javax/management/relation/RoleList.java

```diff
@@ -208,11 +192,9 @@ public void set(int index,
                 IndexOutOfBoundsException {
 
         if (role == null) {
-            // Revisit [cebro] Localize message
-            String excMsg = "Invalid parameter.";
-            throw new IllegalArgumentException(excMsg);
+            throw new IllegalArgumentException("Invalid parameter");
```

Comment:
Removing the trailing dot from "parameter.", here and new line 247 below, to be in line with the rest.